sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1032,3 +1197,276 @@ def _from_generated(cls, score):
     def __repr__(self):
         return "SentimentConfidenceScores(positive={}, neutral={}, negative={})" \
             .format(self.positive, self.neutral, self.negative)[:1024]
+
+
+class EntitiesRecognitionTask(DictMixin):
+    """EntitiesRecognitionTask encapsulates the parameters for starting a long-running Entities Recognition operation.
+
+    :ivar str name: An optional descriptive name for the task.
+    :ivar str model_version: The model version to use for the analysis.
+    :ivar str string_index_type: An optional string for specifying the method used to interpret string offsets.  
```

Comment:
some more context: for the other inputs, we are not exposing `string_index_type` parameter to users (at least for now) since each language has its own set default way of encoding (python uses unicode code points). Once again, not sure if this is a "design change" to make for preview 1, would ask @assafi about it

Reply:
I decided to go ahead and remove this for now.  I know that Assaf wants to have this exposed but we probably want to be consistent about it everywhere, and since the other SDKs are (I think) hiding this for now I will come up with a plan for exposing it in the next preview release (along with other discussed changes for OM).